extension/commercial_data/README.md

```diff
@@ -0,0 +1,398 @@
+```
+
+#### `POST /orders`
+
+Create an order for selected products. Order can contain some additional parameters that specify how the products should be delivered. For example, depending on the collection it might be possible to set the projection, resampling method, bit depth etc of the delivered data.
+
+Backends should expose the available ordering parameters in `/collections/{collection_id}` in the `order_parameters` field, following the `process_parameters` schema of [`GET /service_types`](https://openeo.org/documentation/1.0/developers/api/reference.html#tag/Secondary-Services/operation/list-service-types). 
+
+The request should return the `order_id`, current status and the costs of the order.
+
+```yaml
+schema:
+  title: Create Order Request
+  type: object
+  required:
+    - collection_id
+    - products
+    - parameters
+  properties:
+    collection_id:
+      type: string
+      description: Unique identifier of the collection.
+    products:
+      type: array
+      description: Array of IDs of products to order.
+      items:
+        $ref: 'https://raw.githubusercontent.com/Open-EO/openeo-api/1.0.0/openapi.yaml#/components/schemas/order_id'
+    parameters:
+      type: object
+      description: Key-value pairs of available `order_parameters` as listed at `GET /commercial_data/collections/{collection_id}` for filtering available products.
+      additionalProperties:
+        x-additionalPropertiesName: Order Parameter Name
+        description: Value of the order parameter to be used in the request.
+```
+
+#### `GET /orders/{order_id}`
+
+Get full metadata of the order. The item should follow the [STAC Order Extension](https://github.com/stac-extensions/order), but extended with the spatial and temporal extent information and other metadata about the products.
+
+```json
+{
+  "order:id": "40264b5-c3ae-46f4-a907-0f612d763d97",
+  "order:status": "delivered",
+  "order:date": "2017-01-01T09:32:12Z",
+  "products": [
+    {
+      "type": "Feature",
+      "stac_version": "1.0.0",
+      "id": "c8a1f88d-89cf-4933-9118-45e9c1a5df20",
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              12.36555287044679,
+              41.94403289260048
+            ]
+          ],
+          [
+            12.36571746774068,
+            41.86399361096971
+          ],
+          [
+            12.60746759743069,
+            41.86372776276345
+          ],
+          [
+            12.60758647471871,
+            41.94379931812686
+          ],
+          [
+            12.36555287044679,
+            41.94403289260048
+          ]
+        ]
+      },
+      "properties": {
+        "constellation": "PHR",
+        "acquisitionDate": "2022-03-21T10:11:15.055Z",
+        "azimuthAngle": 179.9852862071639,
+        "cloudCover": 0,
+        "geometryCentroid": {
```

Comment:
Better use proj:centroid

Reply:
I changed all of this according to the suggestions, but I should clarify that this is an example of an Item as it is returned directly by Sentinel Hub, and that different products from different providers have various other fields, which are not present in this example.

Reply:
Okay, then the spec should probably highlight that providers are recommended to align with STAC as much as possible and not only dump existing information "as-is" as then it's not very useful.

Reply:
Added a clarification.